controls/radchartview3d/populating-data/data-binding.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Data Binding
+page_title: Data Binding
+description: Data Binding
+slug: radchartview3d-populating-data-databinding
+tags: axes,3d,chartview,data,binding,itemssource,valuebinding,
+published: True
+position: 3
+---
+
+# Data Binding
+
+This article explains how to bind a 3D chart series to a collection of business objects. To data bound the series you can use its __ItemsSource__ property. The property is of type __IEnumerable__ so you can use most collection types. 
+
+The chart will create a __DataPoint3D__ model for each business object in the ItemsSource collection. To map the properties of the business object to the DataPoint3D you can use the series value binding properties.
+
+* __XValueBinding__: The binding that will be used to fill the data point model XValue property.
+* __YValueBinding__: The binding that will be used to fill the data point model YValue property.
+* __ZValueBinding__: The binding that will be used to fill the data point model ZValue property.
+
+The properties are of type __DataPointBinding__ which is an abstract class that has two implementations - __PropertyNameDataPointBinding__ and __GenericDataPointBinding__.
+
+The __PropertyNameDataPointBinding__ excepts a path (name) to a property in the business object of the data point. The binding class uses a reflection and the property name to get the corresponding value and assing it to the data point model. This type of binding is used when you set the value binding properties in XAML.
```

Comment:
The binding class obtains the property using reflection, gets the corresponding value and assigns it to the data point model.